animation_nodes/data_structures/meshes/mesh_data.pyx

```diff
@@ -92,6 +96,10 @@ cdef class Mesh:
     def getPolygonBitangents(self):
         return calculatePolygonBitangents(self.getPolygonTangents(), self.getPolygonNormals())
 
+    @derivedMeshDataCacheHelper("Linked Vertices")
+    def getLinkedVerticesCache(self):
```

Comment:
You can use the same naming convention used for other attributes.
`getLinkedVertices` -> `calculateLinkedVertices`.
`getLinkedVerticesCache` -> `getLinkedVertices`.

Reply:
Okay.